src/BuiltInTools/BrowserRefresh/Microsoft.AspNetCore.Watch.BrowserRefresh.csproj

```diff
@@ -0,0 +1,16 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
```

Comment:
Entire project finished in sub second, expected? https://helixre8s23ayyeko0k025g8.blob.core.windows.net/dotnet-sdk-refs-pull-15188-merge-4ce2d126a25f4ddab9/Microsoft.AspNetCore.Watch.BrowserRefresh.Tests.dll/console.b290de92.log?sv=2019-07-07&se=2021-01-31T19%3A13%3A22Z&sr=c&sp=rl&sig=FFIwC9n2%2BDZEtMSrBHURe9rTcg1Iul2Q%2FbmZx5PkeI8%3D